composer.json

```diff
@@ -1,54 +1,63 @@
 {
-	"name": "pressbooks/pressbooks-plugin-scaffold",
-	"license": "GPL-3.0-or-later",
-	"type": "wordpress-plugin",
-	"description": "Scaffolding for a Pressbooks plugin.",
-	"homepage": "https://github.com/pressbooks/pressbooks-plugin-scaffold",
-	"authors": [
-		{
-			"name": "Book Oven Inc.",
-			"email": "code@pressbooks.com",
-			"homepage": "https://pressbooks.org"
-		}
-	],
-	"keywords": ["ebooks", "publishing", "webbooks"],
-	"support": {
-		"email": "code@pressbooks.com",
-		"issues":
-			"https://github.com/pressbooks/pressbooks-plugin-scaffold/issues/",
-		"source": "https://github.com/pressbooks/pressbooks-plugin-scaffold/"
-	},
-	"config": {
-		"allow-plugins": {
-			"composer/installers": true,
-			"dealerdirect/phpcodesniffer-composer-installer": true
-		}
-	},
-	"require": {
-		"php": ">=8.1",
-		"composer/installers": "^2.2"
-	},
-	"require-dev": {
-    "phpcompatibility/php-compatibility": "^9.3",
-		"pressbooks/coding-standards": "dev-dev",
+  "name": "pressbooks/pressbooks-plugin-scaffold",
+  "license": "GPL-3.0-or-later",
+  "type": "wordpress-plugin",
+  "description": "A scaffold for Pressbooks plugins.",
+  "homepage": "https://github.com/pressbooks/%PLUGIN_SLUG%",
+  "authors": [
+    {
+      "name": "Book Oven Inc.",
+      "email": "code@pressbooks.com",
+      "homepage": "https://pressbooks.org"
+    }
+  ],
+  "keywords": ["ebooks", "publishing", "webbooks"],
+  "support": {
+    "email": "code@pressbooks.com",
+    "issues": "https://github.com/pressbooks/pressbooks-plugin-scaffold/issues/",
+    "source": "https://github.com/pressbooks/pressbooks-plugin-scaffold/"
+  },
+  "config": {
+    "allow-plugins": {
+      "composer/installers": true,
+      "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
I don't think this is needed anymore now that Pint is being used instead of PHPCodeSniffer

Reply:
@greatislander Done! Thank you, I didn't see it there.